util/topgen/intermodule.py

```diff
@@ -457,6 +486,15 @@ def check_intermodule(topcfg: Dict, prefix: str) -> int:
                 .format(req_struct["width"], rsps_width))
             error += 1
 
+    for item in topcfg["inter_module"]["top"] + topcfg["inter_module"]["ext"]:
+        sig_m, sig_s, sig_i = filter_index(item)
```

Comment:
wait actually, shouldn't some of the earlier checks that apply to `connect` apply here also?